db/schema.rb

```diff
@@ -75,24 +75,6 @@
     t.integer "variant"
   end
 
-  create_table "external_api_keys",
```

Comment:
this seems odd. I think we need to make sure these removals on api is not checked in.

Reply:
This was accidentally committed onto `main` from `develop` [here](https://github.com/bcgov/HOUS-permit-portal/pull/458/files#diff-cbf8b25d6d853acf58fa9057841f407d29ffe90649c75cf9e3f51b2d6e3aa1d3) . The external API keys table only exists on `develop`.